foundry.toml

```diff
@@ -1,6 +1,41 @@
+# Full reference https://github.com/foundry-rs/foundry/tree/master/config
+
 [profile.default]
-src = 'contracts'
-out = 'out'
-libs = ['node_modules', 'lib']
-test = 'test'
-cache_path  = 'cache_forge'
\ No newline at end of file
+  auto_detect_solc = false
+  block_timestamp = 1_680_220_800 # March 31, 2023 at 00:00 GMT
+  bytecode_hash = "none"
+  cbor_metadata = false
+  evm_version = "shanghai"
```

Comment:
Unless you're sure that these contracts aren't going to be deployed on chains that don't support for example PUSH0, I'd use "paris" for now.

For more info, see https://www.evmdiff.com/diff?base=1&target=10

Reply:
Addressed in 700d99b